src/extension.ts

```diff
@@ -81,26 +81,29 @@ export function activate(context: ExtensionContext) {
 								resolve();
 								commands.registerCommand(Commands.START_DEBUG_SESSION, async (config) => {
 									if (config.request === 'launch') {
-										if (!config.startupClass) {
-											vscode.window.showErrorMessage('Please specify startupClass in launch.json first.');
+										if (!config.mainClass) {
+											vscode.window.showErrorMessage('Please configure the main class in launch.json first.');
 											return;
-										} else if (!config.classpath) {
+										} else if (!config.classPaths || !Array.isArray(config.classPaths) || !config.classPaths.length) {
 											const params: ClasspathResolveRequestParams = {
-												startupClass: config.startupClass,
+												mainClass: config.mainClass,
 												projectName: config.projectName,
 											};
-											const classpath = await languageClient.sendRequest(ClasspathResolveRequest.type, params);
-											config.classpath = classpath;
+											config.classPaths = await languageClient.sendRequest(ClasspathResolveRequest.type, params);
```

Comment:
the return value of the command is string?

Reply:
i changed it to string array.